src/README.md

```diff
@@ -239,5 +239,33 @@ juju config trove nova-keypair=trove-keypair
 ```
 
 
+## Troubleshooting
+
+Each database instance deployed has **2 ports** for the:
+- `tenant-net`
+- `trove-net`
+
+The **tenant-net port** has a security group (`trove_sg-uuid`) that allows all egress traffic 
+and ingress traffic specific to the database created.
+
+The **trove-net port** has a security group based on the charm `management-security-groups` config option:
+- If it is not set, the security group will be `trove-sec-group` (`openstack security group list --tag charm-trove`) that allows the RabbitMQ traffic through the `mgmt-net`.
+- If it is set, the security group will be `management-security-group` that allows the traffic through the `mgmt-net` based on the specific rules that are set.
+
+**NOTE**: A `trove_sg-uuid` security group is created for every instance.       
```

Comment:
You have some trailing whitespaces in this section you added, including this line. You'll see them most clearly in ``git show``.

Reply:
Removed them, thanks.